src/LogikUI/Program.cs

```diff
@@ -50,15 +51,99 @@ class Program
             return toolbar;
         }
 
-        static MenuBar CreateMenuBar() {
-            MenuBar bar = new MenuBar();
+        static void AddFilters(FileChooserDialog fcd)
+        {
+            FileFilter filter = new FileFilter();
+            filter.Name = "Project Files";
+            filter.AddPattern("*.xml");
+            filter.AddPattern("*.XML");
```

Comment:
In the future/now we might want to have our own file extension so the files don't get confused with other xml files.